### PR TITLE
CLOUDP-70018: mongocli iam team(s) create

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70018

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

Got this error when running the command without username.
```
./bin/mongocli iam teams create test-Team
Error: POST https://cloud-dev.mongodb.com/api/atlas/v1.0/orgs/5e429e7706822c6eac4d5970/teams: 400 (request "Bad Request") A team must contain at least one user. Add a user to your team to continue.
```

atlas
```
./bin/mongocli iam teams create test-Team --username charts+5e4e593f70dfbf1010295836@mongodb.com

Team 'test-Team' created.
```

om
```
./bin/mongocli iam teams create test-Team --username test@gmail.com -P ops --orgId 5f4f3c760ed5b166b56b02ce

Team 'test-Team' created.
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
waiting for https://github.com/mongodb/mongocli/pull/394

File: internal/cli/iam/teams/create.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.name = args[0]
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringSliceVar(&opts.userNames, flag.Username, []string{}, usage.Username)
```
Comment (gssbzn) [inline internal/cli/iam/teams/create.go:77]: can we have a different `usage.Username` to explain this is a list of members of the team?

**Review by gssbzn — COMMENTED** [review]
Looks great, left a little suggestion to make the usage of the username a bit clearer since this can be a list and not a single user

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**